server/src/controllers/claim-transaction.ts

```diff
@@ -9,6 +9,7 @@ const getTotal = async (req: Request, res: Response, next: NextFunction) => {
     const result = data.map((rec: any) => ({
       total_amount_txs: rec.total_amount_txs || 0,
       count_txs: rec.count_txs || 0,
+      ubi_quota: rec.ubi_quota || 0,
```

Comment:
please try to set default values in the mongoose schema. Probably this allows you to avoid `|| 0` and and you could just
```javascript
 const result = data.map((rec: any) => pick(rec, 'total_amount_txs', 'count_txs', 'ubi_quota', 'date'))
```